.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to Ignite CLI
+
+[![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](http://standardjs.com/)
+
+We welcome all contributors to Ignite Bowser! This contributing guide will help you get up and running to submit your first pull request.
+
+Before submitting a pull request, you will want to make sure that your branch meets the following requirements:
+
+_These examples assume you're using yarn rather than npm. If you're not using yarn, replace these commands with the [appropriate npm alternative](https://shift.infinite.red/npm-vs-yarn-cheat-sheet-8755b092e5cc)_
+
+- Everything works on iOS/Android
+- Jest Tests pass in the root folder (`yarn test`)
+- New tests are included for any new functionality
+- Code is compliant with StandardJS (`yarn lint`)
+- Branch has already been [synced with the upstream repo](https://help.github.com/articles/syncing-a-fork/) and any merge-conflicts have been resolved.
+
+## Requirements
+
+- Node 7.6+
+- NPM 4 (ships with Node 7)
+
+## Getting Started
+
+1. Fork and then clone the repo (`git clone git@github.com:<YOURGITHUBUSER>/ignite-bowser.git`)
+2. Pull all package dependencies (`yarn`)
+3. Make your changes
+
+Test it out:
+
+Test your changes by creating a new boilerplate using [ignite-cli](https://github.com/infinitered/ignite) and pointing it to your newly created boilerplate.
+
+```sh
+$ ignite new <YOUR_BOILERPLATE_NAME> -b /full/path/to/boilerplate
+```
+
+Now you're ready to check out a new branch and get hacking on Ignite Bowser!
+
+
+## How to Build and Run App
+
+Refer to [this guide](https://github.com/infinitered/ignite/blob/master/.github/CONTRIBUTING.md#how-to-build-and-run-app)
+
+## Testing the App
+
+We use [Jest](https://jestjs.io) for testing.
+
+To run tests from the ignite-bowser folder:
+
+```
+$ yarn test
+```
+
+
+## :no_entry_sign: Standard Compliant
+
+[![js-standard-style](https://cdn.rawgit.com/feross/standard/master/badge.svg)](https://github.com/feross/standard)
+
+This project adheres to Standard. Our CI enforces this, so we suggest you enable linting to keep your project compliant during development.
```

Comment:
This whole PR looks really good, @harrisrobin! However, Bowser isn't compliant with Standard - `npx standard` produces a ton of complaints, but `yarn lint` doesn't (because Standard doesn't respect Bowser's package.json's `eslintConfig` overrides).

@jamonholmgren What do you think about deemphasizing the Standard compliance in favor of just talking about how we want `yarn lint` to pass? (We could instead work on Standard compliance - but I think that's a larger discussion, since it'd affect generated apps too, and there seem to be [problems with using Standard on Typescript sources](https://github.com/standard/standard#typescript).)

Reply:
Good catch. I can definitely change the language around that, I'll get to it ASAP.

Reply:
I agree @bryanstearns, that sounds like a great way to handle linting.